src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -404,7 +404,7 @@ func (a *AdminAPI) sendOne(
 // Unfortunately these assumptions do not match all environments in which
 // Redpanda is deployed, hence, we need to reintroduce the sendAll method and
 // broadcast on writes to the Admin API.
-func (a *AdminAPI) sendAll(method, path string, body, into interface{}) error {
+func (a *AdminAPI) sendAll(rootCtx context.Context, method, path string, body, into interface{}) error {
```

Comment:
I think `ctx` here and `childCtx` below would be more consistent. Only worth addressing this nit if there are other comments.